src/include/optimizer/statistics/count_min_sketch.h

```diff
@@ -145,6 +145,21 @@ class CountMinSketch {
    */
   size_t GetTotalCount() const { return total_count_; }
 
+  /**
+   * Clear the sketch
+   */
+  void Clear() { sketch_.clear(); }
+
+  /*
+    Return the sketch
```

Comment:
nit: Does not follow our doxygen comment format.

Reply:
I'll fix the documentation formats and upload a new commit.